docs/setup-guide/cluster_setup.md

```diff
@@ -0,0 +1,422 @@
+	```
+	2. Download the Google Cloud public signing key:
+	```
+	sudo curl -fsSLo /usr/share/keyrings/kubernetes-archive-keyring.gpg https://packages.cloud.google.com/apt/doc/apt-key.gpg
+	```
+	3. Add the Kubernetes apt repository:
+	```
+	echo "deb [signed-by=/usr/share/keyrings/kubernetes-archive-keyring.gpg] https://apt.kubernetes.io/ kubernetes-xenial main" | sudo tee /etc/apt/sources.list.d/kubernetes.list
+	```
+	4. Update apt package index, install kubelet, kubeadm and kubectl, and pin their version:
+	```
+	sudo apt-get update
+	apt-get install -qy kubelet=1.21.1-00 kubectl=1.21.1-00 kubeadm=1.21.1-00
+	sudo apt-mark hold kubelet kubeadm kubectl
+	```
+
+- Next, run the command to enable docker service systemctl enable docker.service
+
+###  1.2.4.	Start a cluster using kubeadm
+- (referring doc: https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/)
+- 1. Run command (it might cost a few minutes)
+```
+kubeadm init
+```
+- 2. At the end of the screen output, you will see info about setting the kubeconfig. Do the following if you are the root user:
+ ```
+ export KUBECONFIG=/etc/kubernetes/admin.conf
+ ```
+- 3. Check the cluster is up by running some commands, like
+```
+kubectl get nodes
+```
+
+
+## 1.3.	Install GoLang Tools to Cloud core and Edge core
+###  1.3.1.	Install GoLang
+- You should in root folder (**copy command line  should by line by line to run**).
+	```
+	GOLANG_VERSION=${GOLANG_VERSION:-"1.14.15"}
+	
+	sudo apt -y update
+	
+	sudo apt -y install make
+	
+	sudo apt -y install gcc
+	
+	sudo apt -y install jq
+	
+	wget https://dl.google.com/go/go${GOLANG_VERSION}.linux-amd64.tar.gz -P /tmp
+	
+	sudo tar -C /usr/local -xzf /tmp/go${GOLANG_VERSION}.linux-amd64.tar.gz
+	```
+	notes: you can use following command see the package list
+	```
+	apt list --upgradable
+	```
+- Instal vim
+```
+sudo apt-get install vim
+```
+###  1.3.2.	Configuration GoLang Path.
+- Open "~/.bashrc" file and add two line to to file end, then save and exit
+```
+vi ~/.bashrc
+
+export PATH=$PATH:/usr/local/go/bin
+export GOPATH=/usr/local/go/bin
+export KUBECONFIG=/etc/kubernetes/admin.conf
+```
+- run following line and let source file effective. The check version and environment value.
+```
+source ~/.bashrc
+
+go version
+
+go env
+```
+###  1.3.3.	Setup project location.
+- create project folder
+```
+mkdir -p go/src/github.com
+```
+- go to project folder
+```
+cd go/src/github.com
+```
+- clone fornax repo, change name to Kubeedge, go to "kubeedge" folder, and compile code by "make all"
+```
+git clone https://github.com/CentaurusInfra/fornax.git
+mv fornax kubeedge
+cd kubeedge
+make all
+```
+- if you want to modify the source code, you need create your practice branch.
+```
+git checkout -b practicebransh
+```
+
+## 1.4.	Fornax Configuration
+###  1.4.1.	Kubecofig File Preparation
+- Copy the admin kubeconfig file of cluster A to machine B, the kubecofig file of cluster B to the machine of cluster C.
```

Comment:
this is the first place in this doc where cluster A and B are mentioned. Reader might be confused like "what A and B mean?", "the setup before this, was that to be done on A or B or both?"

Add a general description here about A-B-C configuration before using term "cluster-A"

Reply:
Add three place:
## 1.1. Virtual Machine Setup and Configuration (We use AWS for example)
-	Ubuntu 18.04, **Create A virtual machine as an cloud-core, Create B and C virtual machine as edge-core.**
## 1.2.	Install Kubernetes Tools to Cloud core and Edge core
**The general steps to install software tools to machine A (machine B and C have the same steps as machine A):**

### 1.3.4    Setup machine B and machine C 
Repeat steps from 1.2.1 to 1.3.3, install all software tools to machine B and C.